rules-java/api/src/main/java/org/jboss/windup/rules/apps/mavenize/bom.xml.ftl

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<#--
+pom: class Pom
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <#-- These constructs are intended for correct formatting of the resulting pom.xml. -->
+    <#assign i1 = ""?left_pad(1*4)>
+    <#assign i2 = ""?left_pad(2*4)>
+    <#assign i3 = ""?left_pad(3*4)>
+    <#assign i4 = ""?left_pad(4*4)>
+
+    <properties>
+    </properties>
+
+    <#if pom.parent??>
+    <parent>
+        <groupId>${pom.parent.coords.groupId}</groupId>
+        <artifactId>${pom.parent.coords.artifactId}</artifactId>
+        <version>${pom.parent.coords.version}</version>
+    </parent>
+    </#if>
+
+    <groupId>${pom.coords.groupId}</groupId>
+    <artifactId>${pom.coords.artifactId}</artifactId>
+    <#if pom.coords.version??>${i1}<version>${pom.coords.version}</version>${"\n"}</#if><#t><#-- Null if same as parent. -->
+    <#if pom.coords.packaging??>${i1}<packaging>${pom.coords.packaging}</packaging>${"\n"}</#if><#t>
+    <#if pom.coords.classifier??>${i1}<classifier>${pom.coords.classifier}</classifier>${"\n"}</#if><#t>
+
+    <#if pom.name??>${i1}<name>${pom.name}</name>${"\n"}</#if><#t>
+    <#if pom.description??>${i1}<description>${pom.description}</description>${"\n"}</#if><#t>
+
+    <dependencyManagement>
+        <dependencies>
+
+            <!-- JBoss distributes a complete set of Java EE APIs including a Bill
+                of Materials (BOM). A BOM specifies the versions of a "stack" (or a collection)
+                of artifacts. We use this here so that we always get the correct versions
+                of artifacts. Here we use the jboss-eap-javaee7 stack (you can
+                read this as the JBoss stack of the Java EE APIs and related components.  -->
+            <dependency>
+                <groupId>org.jboss.bom</groupId>
+                <artifactId>jboss-eap-javaee7</artifactId>
+                <version>7.0.0-build-12</version>
```

Comment:
we should show something different than this BOM.
```
           <dependency>
                <groupId>org.jboss.spec</groupId>
                <artifactId>jboss-javaee-6.0</artifactId>
                <version>${version.org.jboss.spec.jboss.javaee.6.0}</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```
where `version.org.jboss.spec.jboss.javaee.6.0` is something like *3.0.3.Final*

If we want to use extended BOM with JBoss GAVs not only Java EE specific, we should use:
```
           <dependency>
                <groupId>org.jboss.spec</groupId>
                <artifactId>jboss-javaee-6.0-with-all</artifactId>
                <version>1.0.7.Final</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
```
You can use instead of jboss-javaee-6.0-with-all some specific (http://search.maven.org/#search|ga|1|g%3A%22org.jboss.bom%22%20AND%20v%3A%221.0.7.Final%22), but I would not recommend it for our purpose

 At least I can see here a mismatch between EAR maven plugin definition as Java EE 6 and now we provide BOM with Java EE 7. There are similar BOMs for Java EE 7 look at http://search.maven.org/#search|gav|1|g%3A%22org.wildfly.bom%22%20AND%20a%3A%22jboss-javaee-7.0-wildfly%22